src/utils/EventUtils.ts

```diff
@@ -1,17 +1,18 @@
 import { Result } from "@ethersproject/abi";
-import { delay } from "./common";
-import { SortableEvent } from "../interfaces";
+// eslint-disable-next-line no-restricted-imports
 import { Contract, Event, EventFilter } from "ethers";
+import { Log, SortableEvent } from "../interfaces";
+import { delay } from "./common";
 
 const maxRetries = 3;
 const retrySleepTime = 10;
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function spreadEvent(args: Result = {} as Result): any {
+export function spreadEvent(args: Result | Record<string, any>): { [key: string]: any } {
```

Comment:
nit: we should probably use `unknown` in lieu of `any`

Reply:
Good call: https://github.com/across-protocol/sdk/pull/738/commits/6df63d0c688dd7f5563175df6e0046439fbc7c3a